modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -835,7 +835,7 @@ class CV_EXPORTS GeneralizedHoughBallard : public GeneralizedHough
     virtual int getVotesThreshold() const = 0;
 };
 
-//! Guil, N., González-Linares, J.M. and Zapata, E.L. (1999). Bidimensional shape detection using an invariant approach. Pattern Recognition 32 (6): 1025-1038.
+//! Guil, N., Gonzalez-Linares, J.M. and Zapata, E.L. (1999). Bidimensional shape detection using an invariant approach. Pattern Recognition 32 (6): 1025-1038.
 //! Detects position, translation and rotation
```

Comment:
It would be nice to add reference in doc/opencv.bib file and use it here via `@cite NGuil1999`.